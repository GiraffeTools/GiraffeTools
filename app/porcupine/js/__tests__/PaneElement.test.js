import React from 'react'
import PaneElement from "../paneElement";
import {shallow} from "enzyme";

describe('<PaneElement>', () => {
    it('wraps children in a draggable div', () => {
        const comp = shallow(
            <PaneElement category={[]} id="1">
                {"title"}
            </PaneElement>
        )

        expect(comp).toMatchSnapshot()
    })
})