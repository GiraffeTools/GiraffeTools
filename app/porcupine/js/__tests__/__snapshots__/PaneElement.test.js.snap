// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PaneElement> wraps children in a draggable div 1`] = `
<div
  className="btn btn-block drowpdown-button"
  draggable="true"
  id="1"
  onDragStart={[Function]}
>
  title
</div>
`;
